### ISSUE TITLE
GetRef() in support package should remove 'vendor' path 

### ISSUE THREAD

**Issue by yxuco** [project-flogo/core#224]
**Current behavior (how does the issue manifest):**
When working with older third-party packages that uses /vendor packages, GetRef() returns ref with path including '.../vendor/'

**Expected behavior:**
To make it work, GetRef() should return only the ref name, excluding the prefix up to '/vendor/'.

**Minimal steps to reproduce the problem (not required if feature enhancement):**

**Please tell us about your environment (Operating system, docker version, browser & web ui version, etc):**
OSX

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 0.X.X
1.1.0

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by lixingwang** [comment]
@yxuco GetRef most focusing on getting Flogo contribution ref.   Are you saying that you put flogo contributions under the vendor?

**Comment by yxuco** [comment]
Yes, I am compiling Flogo flow as a third-party app (i.e., Hyperledger Fabric v1.4 chaincode), which does not support go module, and so flogo packages are included under vendor folder, and so GetRef() would return the vendor folder path, causing runtime errors.
